scripts/pylint_extensions.py

```diff
@@ -41,6 +41,11 @@ class ExplicitKeywordArgsChecker(checkers.BaseChecker):
             'non-explicit-keyword-args',
             'All keyword arguments should be explicitly named in function call.'
         ),
+        'C0007': (
```

Comment:
A couple of thoughts here:
1. Any specific reason for choosing the code as `C0007`? Ideally, we should follow the series and name it as `C0002`.
2. Can you re-check the message `Keyword argument %s are not to be named explicitly in %s call of %s.`? The grammar here should be singular.

Reply:
Ho @apb7! Thanks for the review, I'll make the requisite changes and commit soon.
1.All codes up till C0006 are in use but other checkers in the same file, and hence I went for the next available one
2. I'll make the changes